GDAXClient/Services/Products/ProductsService.cs

```diff
@@ -64,5 +65,16 @@ public class ProductsService : AbstractService
 
             return productStatsResponse;
         }
+        
+       
+        public async Task<IEnumerable<object[]>> GetProductHistoryAsync(ProductType productPair, DateTime start, DateTime end, int granularity)
```

Comment:
perhaps this can be named `GetHistoricRatesAsync` to match the docs. also this should be added to readme